settings.gradle

```diff
@@ -29,6 +29,7 @@ include 'clients',
     'log4j-appender',
     'metadata',
     'raft',
+    'shell',
```

Comment:
I liked @mumrah's suggestion to call this module `metashell`.

Reply:
It would be nice to have it as just "shell" until we get another shell....

Reply:
Ok. I was mainly objecting to the generality of the name since this it is focused only on the metadata for KIP-500.